dockstore-integration-testing/src/test/java/io/dockstore/webservice/UserResourceIT.java

```diff
@@ -274,10 +280,21 @@ public void testSelfDestruct() throws ApiException {
             adminWorkflowsApi.getWorkflowByPath(WorkflowIT.DOCKSTORE_TEST_USER2_HELLO_DOCKSTORE_WORKFLOW, null, false);
 
         } catch (ApiException e) {
+            assertTrue(e.getCode() == HttpStatus.SC_BAD_REQUEST);
```

Comment:
assertEquals(expected, actual) or assertEquals(message, expected, actual) here and below